infra/main.bicep

```diff
@@ -0,0 +1,421 @@
+    eventGridSystemTopicName: eventGridSystemTopicName
+    storageAccountId: storage.outputs.StorageAccountId
+    queueName: storage.outputs.StorageAccountName_default_doc_processing_name
+    blobContainerName: blobContainerName
+  }
+}
+
+module hostingplan './core/host/appserviceplan.bicep' = {
+  name: hostingPlanName
+  scope: rg
+  params: {
+    name: hostingPlanName
+    location: location
+    sku: {
+      name: hostingPlanSku
+    }
+    kind: 'linux'
+    reserved: true
+  }
+}
+
+module web './app/web.bicep' = {
+  name: websiteName
+  scope: rg
+  params: {
+    name: websiteName
+    location: location
+    tags: { 'azd-service-name': 'web' }
+    appServicePlanId: hostingPlanName
+    // storageAccountName: storageAccountName
+    applicationInsightsName: monitoring.outputs.applicationInsightsName
+    azureOpenAIName: openai.outputs.name
+    azureCognitiveSearchName: azureCognitiveSearch
+    // formRecognizerName: formRecognizerName
+    // contentSafetyName: contentSafetyName
+    appSettings: {
+      AZURE_SEARCH_SERVICE: 'https://${azureCognitiveSearch}.search.windows.net'
+      AZURE_SEARCH_INDEX: azureSearchIndex
+      AZURE_SEARCH_CONVERSATIONS_LOG_INDEX: azureSearchConversationLogIndex
+      AZURE_SEARCH_SEMANTIC_SEARCH_CONFIG: azureSearchSemanticSearchConfig
+      AZURE_SEARCH_INDEX_IS_PRECHUNKED: azureSearchIndexIsPrechunked
+      AZURE_SEARCH_TOP_K: azureSearchTopK
+      AZURE_SEARCH_ENABLE_IN_DOMAIN: azureSearchEnableInDomain
+      AZURE_SEARCH_CONTENT_COLUMNS: azureSearchContentColumns
+      AZURE_SEARCH_FILENAME_COLUMN: azureSearchFilenameColumn
+      AZURE_SEARCH_TITLE_COLUMN: azureSearchTitleColumn
+      AZURE_SEARCH_URL_COLUMN: azureSearchUrlColumn
+      AZURE_OPENAI_RESOURCE: azureOpenAIResource
+      AZURE_OPENAI_MODEL: azureOpenAIModel
+      AZURE_OPENAI_MODEL_NAME: azureOpenAIModelName
+      AZURE_OPENAI_TEMPERATURE: azureOpenAITemperature
+      AZURE_OPENAI_TOP_P: azureOpenAITopP
+      AZURE_OPENAI_MAX_TOKENS: azureOpenAIMaxTokens
+      AZURE_OPENAI_STOP_SEQUENCE: azureOpenAIStopSequence
+      AZURE_OPENAI_SYSTEM_MESSAGE: azureOpenAISystemMessage
+      AZURE_OPENAI_API_VERSION: azureOpenAIApiVersion
+      AZURE_OPENAI_STREAM: azureOpenAIStream
+      AZURE_OPENAI_EMBEDDING_MODEL: azureOpenAIEmbeddingModel
+      AZURE_FORM_RECOGNIZER_ENDPOINT: 'https://${location}.api.cognitive.microsoft.com/'
+      AZURE_BLOB_ACCOUNT_NAME: storageAccountName
+      AZURE_BLOB_CONTAINER_NAME: blobContainerName
+      ORCHESTRATION_STRATEGY: orchestrationStrategy
+      AZURE_CONTENT_SAFETY_ENDPOINT: 'https://${location}.api.cognitive.microsoft.com/'
+      AZURE_BLOB_ACCOUNT_KEY: storage.outputs.AZURE_BLOB_ACCOUNT_KEY
+      APPINSIGHTS_CONNECTION_STRING: monitoring.outputs.applicationInsightsConnectionString
+      AZURE_FORM_RECOGNIZER_KEY: resources.outputs.AZURE_FORM_RECOGNIZER_KEY
+      AZURE_CONTENT_SAFETY_KEY: resources.outputs.AZURE_CONTENT_SAFETY_KEY
+    }
+  }
+  dependsOn:[
```

Comment:
You shouldn't need dependsOn if you are referencing the modules by the reference, not the name.